graphbolt/include/csc_sampling_graph.h

```diff
@@ -16,22 +16,43 @@ namespace sampling {
 using StringList = std::vector<std::string>;
 
 /**
- * @brief Heterogeneous information in the Graph.
+ * @brief Structure representing heterogeneous information about a graph.
+ *
+ * Example usage:
+ *
+ * HeteroInfo info({"A", "B", "C"}, {"X", "Y", "Z"}, node_offset, edge_types);
+ *
+ * This example creates a `HeteroInfo` object with three node types ("A", "B",
+ * "C") and three edge types ("X", "Y", "Z"). The `node_offset` tensor
+ * represents the offset array of node type, and the `edge_types` tensor
```

Comment:
How does the `node_offset` tensor represents the offset array of node type?

Give an example?

Reply:
sure, add the example